engine/src/main/java/org/terasology/config/AudioConfig.java

```diff
@@ -20,49 +20,89 @@
 import java.beans.PropertyChangeSupport;
 
 /**
+ * For configuring the audio of the game
  */
 public class AudioConfig {
     public static final String SOUND_VOLUME = "soundVolume";
     public static final String MUSIC_VOLUME = "musicVolume";
 
-    private float soundVolume;
-    private float musicVolume;
-    private boolean disableSound;
+    private float soundVolume;          // The sound volume
```

Comment:
No need to comment all private fields.
In some cases it makes sense to explain a field but `soundVolume` is pretty self-explanatory right? :)